### PR TITLE
support wasm, delay

### PR DESCRIPTION
**Pull request by RogerKSI**
## Description
- support ignore_ack
- support wasm
- support delay

Note: **get from master and resolve the conflict**

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules).
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).
- [ ] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: config_example.toml [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# The global section has parameters that apply globally to the relayer operation.
+[global]
+
+# Specify the verbosity for the relayer logging output. Default: 'info'
+# Valid options are 'error', 'warn', 'info', 'debug', 'trace'.
+log_level = 'trace'
+
+
```
Comment (taobun) [inline config_example.toml:8]: Should we add `packet-delay`?

**Review by taobun — CHANGES_REQUESTED** [review]

File: config_example.toml (hunk 1 above)
Comment (colmazia) [reply config_example.toml:8]: Yes, i forgot sorry

File: crates/relayer/src/event/rpc.rs [hunk 2]
```diff
@@ -116,6 +116,7 @@ use super::{ibc_event_try_from_abci_event, IbcEventWithHeight};
 pub fn get_all_events(
     chain_id: &ChainId,
     result: RpcEvent,
+    ignore_port_channel: &Vec<PortChannelId>,
```
Comment (colmazia) [inline crates/relayer/src/event/rpc.rs]: Why no clippy check fail here

Comment (colmazia) [reply crates/relayer/src/event/rpc.rs]: I found that there is no clippy anymore in the new version

**Comment by colmazia** [comment]
Tested in Mac for single Hermer

**Pull request merged**